tests/e2e/support/objects/app-files/share/actions.ts

```diff
@@ -187,10 +194,25 @@ export const hasPermissionToShare = async (
   return !(await page.isVisible(noPermissionToShareLabel))
 }
 
-export const copyQuickLink = async (args: copyLinkArgs): Promise<void> => {
+export const copyQuickLink = async (args: copyLinkArgs): Promise<string> => {
   const { page, resource, via } = args
+  let url = ''
+  const linkName = 'Quicklink'
+
   if (via === 'CONTEXT_MENU') {
     await clickActionInContextMenu({ page, resource }, 'create-quicklink')
-    await waitForPopupNotPresent(page)
   }
+
+  if (config.backendUrl.startsWith('https')) {
+    url = await page.evaluate(() => navigator.clipboard.readText())
```

Comment:
Just to clarify. function `navigator.clipboard.readText()` doesn't work for http?
in oc10 where we use http quick link copies to clickboard

Reply:
yeah, `navigator.clipboard` is only available for secure context (i.e. `https`). Also, see [comment](https://github.com/owncloud/web/pull/8499#discussion_r1116562349), [Clipboard](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard)